src/media/js/main.js

```diff
@@ -32,7 +32,7 @@ console.log('Marketplace Communication Dashboard');
         nunjucks.env.dev = true;
         var nunjucks_globals = require('nunjucks').require('globals');
         nunjucks_globals.gravatar = function(gravatar_hash, size) {
-            return ('http://www.gravatar.com/avatar/' + gravatar_hash + '?s=' +
+            return ('https://www.gravatar.com/avatar/' + gravatar_hash + '?s=' +
```

Comment:
Though it seems to be working with `www`, gravatar seems to recommend using `https://secure.gravatar.com/` (http://en.gravatar.com/site/implement/images/ at the end)